webpackfile.js

```diff
@@ -0,0 +1,56 @@
+const path = require('path');
```

Comment:
Maybe de-structure this?

```suggestion
const { resolve } = require('path');
```

Reply:
Yeah, that's a good point. Updated in 4297187